deploy/inference-gateway/epp/Makefile

```
@@ -144,6 +144,27 @@ image-multiarch: ## Build multi-arch image (requires --push ; --load not support
 image-multiarch-push: PUSH=--push ## Build and push multi-arch image to registry
 image-multiarch-push: image-multiarch
 
+# Export the Go compliance SBOM + harvested license texts to SBOM_DEST as plain
+# host files (no image push/load). Single-platform (the SBOM is arch-independent
+# JSON + license texts) and reuses the warm build cache from a preceding
+# image-multiarch[-push] — the build args mirror image-multiarch exactly so the
+# go-builder stage cache-hits. The frontend build then reads these from its build
+# context instead of re-pulling the pushed EPP image (see Dockerfile sbom-export).
+SBOM_DEST ?= $(DYNAMO_DIR)/.epp-sbom
+.PHONY: sbom-export
+sbom-export: ## Export Go SBOM + license texts to SBOM_DEST (reuses build cache)
```

checkmake (0.3.2)

[warning] Target body for "sbom-export" exceeds allowed length of 5 lines (11).

(maxbodylength)